src/include/xlist.h

```diff
@@ -63,7 +63,7 @@ class xlist {
 
 private:
   item *_front, *_back;
-  int _size;
+  uint64_t _size;
```

Comment:
I would think that this should be size_t rather than uint64_t -- no need to burden someone on a 32 bit arch with a 64 bit size field.

Reply:
Makes sense, changed!